mono/metadata/object-internals.h

```diff
@@ -393,7 +393,7 @@ struct _MonoInternalThread {
 	gpointer unused3;
 	gunichar2  *name;
 	guint32	    name_len;
-	guint32	    state;
+	MonoThreadState state;
```

Comment:
Being a bit field, maybe less convincing vs. "strict" enums. The language should support these better.

Reply:
This change is optional.